lib/fictium/exporters/api_blueprint_exporter/action_formatter.rb

```diff
@@ -0,0 +1,31 @@
+module Fictium
+  class ApiBlueprintExporter
+    class ActionFormatter < Fictium::ApiBlueprintExporter::BaseFormatter
+      protected
+
+      def format_sections(action)
+        [build_header(action), build_examples(action)]
+      end
+
+      private
+
+      def build_header(action)
+        result = "## #{action.summary} [#{action.method.to_s.upcase} #{action.path}]"
+        result += "\n\n#{action.description}\n" if action.description.present?
+        result
```

Comment:
`result += action.description.present? ? "\n\n#{action.description}\n" : ''`

Reply:
i like one liners

Reply:
Is also one liner and won't use `+=`

```rb
"#{result}#{action.description.present? ? "\n\n#{action.description}" : ''}"
```